include/VCF_Tasks.h

```diff
@@ -1,69 +1,88 @@
+ *     // Setup code here
+ *     return true;
+ * }
+ * 
+ * void run_example_task() {
+ *     // Task execution code here
+ * }
+ * 
+ * TsTask example_task(1000, TASK_FOREVER, &run_example_task, &task_scheduler, false, &init_example_task);
  */
 
 /* -------------------------------------------------- */
-/*                   TASK PRIORITIES                  */
+/*                   TASK INTERVALS                   */
 /* -------------------------------------------------- */
 /*
- * (1-10) - Car-critical functions (watchdog, shutdown circuit)
- * (11-100) - Performance-critical functions (reading pedals, interfacing w/ inverters, etc)
- * (100+) - Telemetry, logging functions (could be idle functions, too)
- */
-
-#ifndef VCF_TASKS
-#define VCF_TASKS
-
-
-/* From shared_firmware_types library */
-#include "SharedFirmwareTypes.h"
-
-
-
-/**
- * The read_adc1 task will command adc1 to sample all eight channels, convert the outputs, and
- * store them in structs defined in shared_firmware_types. This function relies on adc_1 being
- * defined in VCFGlobals.h.
+ * Performance monitoring: 10-100ms (10-100Hz)
+ * Status updates, non-critical functions: 100ms+ (<10Hz)
  */
-bool init_adc_task();
-bool run_read_adc1_task();
 
-/**
- * NOTE: These channels are UNUSED BY DEFAULT and exist ONLY FOR TESTING. You may edit this
- * manually to add sensors.
- * 
- * The read_adc2 task will command adc2 to sample all eight channels, convert the outputs, and
- * store them in a struct defined in shared_firmware_types. This function relies on adc_2 being
- * defined in VCFGlobals.h.
- */
-bool run_read_adc2_task();
-
-
-/**
- * The buzzer_control task will control the buzzer control pin. This function relies on the 
- * buzzer_control pin definition in VCF_Constants.h;
- */
-bool init_buzzer_control_task();
-bool run_buzzer_control_task();
-
-
-/**
- * The read_gpio task will read the GPIO pins and store the results in a struct defined in 
- * shared_firmware_types. This function relies on the GPIO pins being defined in VCFConstants.h.
- */
-bool init_read_gpio_task();
-bool run_read_gpio_task();
+ #ifndef VCF_TASKS
+ #define VCF_TASKS
+ 
+ 
+ /* From shared_firmware_types library */
+ #include "SharedFirmwareTypes.h"
+ 
 
-#endif /* VCF_TASKS */
\ No newline at end of file
+ 
+ /**
+  * The read_adc1 task will command adc1 to sample all eight channels, convert the outputs, and
+  * store them in structs defined in shared_firmware_types. This function relies on the create funtion
+  * of the acd singleton being called.
+  */
+ bool init_adc_task();
+ void run_read_adc1_task();
+ 
+ 
+ 
+ /**
+  * NOTE: These channels are UNUSED BY DEFAULT and exist ONLY FOR TESTING. You may edit this
+  * manually to add sensors.
+  * 
+  * The read_adc2 task will command adc2 to sample all eight channels, convert the outputs, and
+  * store them in a struct defined in shared_firmware_types. This function relies on the create funtion
+  * of the acd singleton being called.
```

Comment:
nit: when referring to the create() function, write it with the parenthesis to indicate it as a function